DQM/L1TMonitorClient/python/L1TdeStage2uGMT_cfi.py

```diff
@@ -0,0 +1,26 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
@z4027163 why did you clone the configuration for the uGMT DQM that is already in  DQM/L1TMonitor/python/L1TdeStage2uGMT_cfi.py ?
The same question goes for L1TdeStage2CaloLayer1_cfi.py